packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -7,21 +7,21 @@ const DEFAULT_THUMB_SIZE = 12;
 
 export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
   const { theme, className, disabled, checked, inlineLabel, onOffMissing } = props;
-  const { semanticColors, palette } = theme;
+  const { semanticColors } = theme;
 
   // Tokens
   const pillUncheckedBackground = semanticColors.bodyBackground;
   const pillCheckedBackground = semanticColors.inputBackgroundChecked;
   // TODO: after updating the semanticColors slots mapping this needs to be semanticColors.inputBackgroundCheckedHovered
-  const pillCheckedHoveredBackground = palette.themeDark;
-  const thumbUncheckedHoveredBackground = palette.neutralDark;
+  const pillCheckedHoveredBackground = semanticColors.primaryButtonBackgroundPressed;
+  const thumbUncheckedHoveredBackground = semanticColors.actionLinkHovered;
   const pillCheckedDisabledBackground = semanticColors.disabledBodySubtext;
   const thumbBackground = semanticColors.smallInputBorder;
   const thumbCheckedBackground = semanticColors.inputForegroundChecked;
   const thumbDisabledBackground = semanticColors.disabledBodySubtext;
   const thumbCheckedDisabledBackground = semanticColors.disabledBackground;
   const pillBorderColor = semanticColors.smallInputBorder;
-  const pillBorderHoveredColor = semanticColors.inputBorderHovered;
+  const pillBorderHoveredColor = semanticColors.actionLinkHovered;
```

Comment:
@phkuo  can you please advise here? I see semanticColors abused, probably because the colors are wrong?

@Jahnp fyi. We should use input semantic colors, not links for the toggle. is this a design concern? Are the colors wrong?

Reply:
I agree that this is a concern. How useful are semantic colors if they are co-opted by random controls. inputBorderHovered seems to be the most semantic option here. Why can't we use that color? 

@marygans 
Can we rather change inputBorderHovered to rgb(32,31,30) instead?

Reply:
Yes, you shouldn't mix and match multiple families of semantic colors in a single element. It should use input colors like the rest of the component.

Reply:
Its due to the toolkit mismatch
So, I can make the update for the `inputBorderHovered` semantic slot to be changed from `neutralPrimary` to `neutralDark` but the components referencing `inputBorderHovered`(e.g. checkbox, spinButton, and others) would have to be updated to a different semantic slot representing` neutralPrimary` as well. Is this the best approach?

Reply:
Questions like these need to be brought back to the designer. The dev tells the designer that they (the dev) believes there's an inconsistency with the existing redlines, where one component isn't matching the way other components are styled.

(Presumably we will fix this friction once we get designers to start using semantic colors.)

Reply:
Why all inputs are not using the same color? Why Toggle needs to use `neutralDark` and the rest of the inputs `neutralPrimary`. It feels like a bug in the toolkit with the Toggle.

Reply:
I will check with @betrue-final-final on this. Here is a [reference link ](https://github.com/OfficeDev/office-ui-fabric-react/issues/6712)that might be helpful too :)

Reply:
@phkuo What is the right response for a designer here? I don't think there's a right answer that scales. Add a Toggle category of colors is what I would expect them to say. And basically keep doing that for every control because they all have unique requirements.

My theory here is that if there were a programatic definition of shades based on a bg base, we could say which programatic definition to use, and if there isn't one, create one. Basically take the "category" out of the variable names. (Recipes)

Reply:
If the designer is well versed in semantic slots, they can give the appropriate ones to the dev.
Otherwise the designer says what family the toggle is supposed to be. For example, they could say it uses the standard button colors in the off state, and uses primary button colors in the on state. The dev can infer from there which slots to use.

A lot of the confusion here happened because "the redline says this border should be neutralDark but the correct semantic slot here has it as neutralPrimary". If the dev goes off hunting for a neutralDark semantic slot, that completely defeats the purpose of semantic slots. The correct response here is acknowledging that design is trying to do two conflicting things:
1) We're trying to insure consistency across the product. Therefore the Toggle should belong to X family of components and share the same colors as them.
2) But the designer is trying to set a bespoke color there.
So either all components in the X family need to have their borders updated, or the designer gave the wrong color. It's all about consistency. A primary button in one place in the product should share the same colors as a primary button anywhere else in the product. A checkbox in one place should share the same colors as a radio button in another place.

This is just the system revealing errors in the redlines.

Reply:
I closed this because I spoke to Ben and confirmed to stick with neutralPrmary. I agree that there has been a lot of confusion in regards to toolkit vs what's in code. I am documenting these discrepancies and adding it the backlog of the toolkit design workstream backlog to ensure these changes are reflected in the toolkits ASAP and prevent this confusion in the future.